### ISSUE TITLE
Huge Training Loss after a certain number of epochs

### ISSUE THREAD

**Issue by aritra3520** [gudovskiy/cflow-ad#36]
While training with the wide resnet_50 on different surveilliance camera data, I am experiencing a huge loss.
What can be the reason for such outcome?

It will be really helpful if you can contribute to the problem.

**Comment by gudovskiy** [comment]
@aritra3520 may be your data statistics (mu/std normalization) is very different?
